src/main/common/components/ccj/paid-amount.njk

```diff
@@ -22,9 +22,7 @@
 
           <div class="panel panel-border-narrow js-hidden" id="option-yes">
 
-            <p>{{ t('Total claim amount: {{ totalAmount }}', { totalAmount: totalAmount | numeral }) }}</p>
-
-            <input type="hidden" name="claimedAmount" value="{{totalAmount}}" />
+            <input type="hidden" name="claimedAmount" value="{{ totalAmount }}" />
```

Comment:
Why do we have a hidden input on this page?

Reply:
validation, you can't do more than total amount